packages/front-end/components/Features/FeaturesOverview.tsx

```diff
@@ -215,7 +215,14 @@ export default function FeaturesOverview({
   const { jsonSchema, validationEnabled, schemaDateUpdated } = getValidation(
     feature
   );
-  const requireReviews = !!settings?.requireReviews;
+  const requireReviews = checkIfRevisionNeedsReview({
+    feature,
+    baseRevision: revisions.find(
+      (r) => r.version === revision?.baseVersion
+    ) as FeatureRevisionInterface,
```

Comment:
Remove the `as` here, it's hiding a legitimate error.  This might be undefined if the base revision can't be found for some reason and we need to make sure we handle that throughout the code.